lib/container.js

```diff
@@ -283,8 +284,29 @@ function loadGherkinSteps(paths) {
   global.After = fn => event.dispatcher.on(event.test.finished, fn);
   global.Fail = fn => event.dispatcher.on(event.test.failed, fn);
 
-  for (const path of paths) {
-    loadSupportObject(path, `Step Definition from ${path}`);
+  //If gherkin.steps is string, then this will iterate through that folder and send all step def js files to loadSupportObject
+  //If gherkin.steps is Array, it will go the old way
+  //This is done so that we need not enter all Step Definition files under config.gherkin.steps
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected exception block, space or tab after '//' in comment. (spaced-comment)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4548869)